webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -70,24 +117,35 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
             useAgentUser: !!agentUid && !!agentUser,
           });
         }
+        const bindingsList = Loadable.getOrElse({}, namespaceBindingsList);
+        for (const key in bindingsList) {
+          form.setFieldValue(key, bindingsList[key]);
+        }
```

Comment:
re: above, nesting the bindings allows us to directly set the field values instead of looping via:
```suggestion
        namespaceBindingsList.forEach((bindingsList) => form.setFieldValue('binding', {...bindingsList}));
```

note that the object is shallowly cloned because the form object will mutate the object passed to it.